strawberry/field.py

```diff
@@ -34,6 +34,10 @@
 _RESOLVER_TYPE = Union[StrawberryResolver, Callable]
 
 
+def default_resolver(name: str, source: Any, *args, **kwargs):
+    return getattr(source, name)
```

Comment:
Can this be `StrawberryResolver`? Potentially constructed using a `classmethod`?

Reply:
I guess it can be though what would the benefit be?

Reply:
So that `Field.get_result` always deals with the same datatype

Reply:
Ok I see. Will change.